crates/isograph_compiler/src/isograph_literals.rs

```diff
@@ -47,14 +42,14 @@ pub(crate) fn read_files_in_folder(
                 .expect("Expected path to be stringable")
                 .contains("__isograph")
         })
-        .map(|path| read_file(path, canonicalized_root_path))
+        .map(|path| read_file(path, current_working_directory))
         .collect()
 }
 
 pub fn read_file(
     path: PathBuf,
-    canonicalized_root_path: &Path,
-) -> Result<(PathBuf, String), BatchCompileError> {
+    current_working_directory: CurrentWorkingDirectory,
+) -> Result<(RelativePathToSourceFile, String), BatchCompileError> {
     // This isn't ideal. We can avoid a clone if we changed .map_err to match
     let path_2 = path.clone();
```

Comment:
nit: we can remove this clone, and just clone in the the `map_err` fn. I can't make that change in main because it causes conflicts :joy: